main/index.html

```diff
@@ -233,6 +213,16 @@ <h1 class="contact-title">CONTACT</h1>
         </script>
         <script src="https://www.google-analytics.com/analytics.js" async defer></script>
         <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js"></script>
+        <script type="text/javascript">
```

Comment:
lint this ?

Reply:
tried on jshint to check for warnings, it returned 0 warnings.